pkgs/servers/monitoring/alertmanager-executor/default.nix

```diff
@@ -0,0 +1,28 @@
+{ lib, buildGoModule, fetchFromGitHub }:
+
+buildGoModule rec {
```

Comment:
`ldflags` are missing.

Reply:
What should they contain? Nixpkgs manual nor prometheus-am-executor README contain no clues.

Reply:
Usually, a bare minimum for Nixpkgs is `[ "-s" "-w" ]` to strip executable. Since upstream doesn't specify additional flags, you don't have to add more.

https://github.com/NixOS/nixpkgs/blob/master/doc/languages-frameworks/go.section.md#ldflags-var-go-ldflags

Reply:
Fixed in [935b191](https://github.com/NixOS/nixpkgs/pull/189119/commits/935b1917f9d80188da2ecb54bad2a50f86937404)